SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -269,6 +270,50 @@ void USpatialPackageMapClient::ClearRemovedDynamicSubobjectObjectRefs(const Work
 	}
 }
 
+void USpatialPackageMapClient::RemoveBNetLoadOnClientRuntimeRemovedComponents(const Worker_EntityId& EntityId,
+																			  const TArray<SpatialGDK::ComponentData>& NewComponents,
+																			  const SpatialGDK::ActorSystem& ActorSystem
+
+)
+{
+	auto ContainedInComponentsArr = [this, &NewComponents, &EntityId](const FUnrealObjectRef CheckComponentObjRef) {
```

Comment:
```suggestion
	auto ContainedInComponentsArr = [this, &NewComponents, &EntityId](const FUnrealObjectRef& CheckComponentObjRef) {
```
Super-small optimization maybe